core/bn.js

```diff
@@ -88,12 +88,15 @@ sjcl.bn.prototype = {
   /**
    * Convert to a hex string.
    */
-  toString: function() {
-    this.fullReduce();
-    var out="", i, s, l = this.limbs;
-    for (i=0; i < this.limbs.length; i++) {
+  toString: function(doTrim) {
+    var i, s, l, out = new sjcl.bn(this);
+    out.fullReduce();
```

Comment:
I am pretty sure that fullReduce also has the same effect as `trim()` or am I missing something hereß